react/Field/Readme.md

```diff
@@ -19,6 +19,35 @@ Like `Input` component, it can have the following properties:
 </form>
 ```
 
+- `inputRef`
+
+##### Example
+
+```
```

Comment:
Can you add a description of prop ?

Reply:
Does

> This property is mapped to `<Input />` component `inputRef` property.
It gives access to the underlying `<input />` element, for example to give focus or move the caret.

suits you ?

Reply:
Perfect.